From c763c84a6240a0dd07fd971e048d8a3914b37865 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 7 Feb 2018 19:20:38 +0800
Subject: [PATCH] media/video/rk30_camera_oneframe: fix compile warning

drivers/media/video/rk30_camera_oneframe.c:2212:5: warning: this if
clause does not guard... [-Wmisleading-indentation]

Change-Id: Ie304d39082a84b322a15b8b74f5b24d9d394b4d7
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/media/video/rk30_camera_oneframe.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/media/video/rk30_camera_oneframe.c b/drivers/media/video/rk30_camera_oneframe.c
index 337c3df97d85..d13f3c6184d7 100644
--- a/drivers/media/video/rk30_camera_oneframe.c
+++ b/drivers/media/video/rk30_camera_oneframe.c
@@ -2208,9 +2208,9 @@ static int rk_camera_get_formats(struct soc_camera_device *icd, unsigned int idx
 		return 0;
 	}
 
-    ret = rk_camera_try_bus_param(icd, fmt->bits_per_sample);
-    if (ret < 0)
-        return 0;
+	ret = rk_camera_try_bus_param(icd, fmt->bits_per_sample);
+	if (ret < 0)
+		return 0;
 
 	switch (code) {
 		case MEDIA_BUS_FMT_UYVY8_2X8:
-- 
2.35.3

